infra/ansible/playbooks/operator.yaml

```diff
@@ -1,47 +1,181 @@
-- import_playbook: setup.yaml
-- import_playbook: go.yaml
-- import_playbook: rust.yaml
-- import_playbook: eigenlayer-cli.yaml
+- name: Run setup playbook
+  ansible.builtin.import_playbook: setup.yaml
+  vars:
+    host: operator
```

Comment:
For a future improvement: parametrize this variable so it's not repeated so much